File: API/Entities/Enums/ReaderMode.cs
using System.ComponentModel;

namespace API.Entities.Enums;

public enum ReaderMode
{
    [Description("Left and Right")]
    LeftRight = 0,
    [Description("Up and Down")]
    UpDown = 1,
    [Description("Webtoon")]
    Webtoon = 2
}

File: UI/Web/src/app/_models/preferences/reader-mode.ts
/**
 * The pagination method used by the reader
 */
export enum ReaderMode {
    /**
     * Manga default left/right to page
     */
    LeftRight = 0,
    /**
     * Manga up and down to page
     */
    UpDown = 1,
    /**
     * Webtoon reading (scroll) with optional areas to tap
     */
    Webtoon = 2
}